### ISSUE TITLE
ergoCub 1.1 S/N:001 – ergoCubHead computer not turning on and not reachable

### ISSUE THREAD

**Issue by SimoneMic** [robotology/icub-tech-support#1758]
### Robot Name 🤖

ergoCub 1.1 S/N:001

### Request/Failure description

The head computer on ergoCubSN001 is not reachable and the head display is not turning on. Probably the computer on the head is not powered.

### Detailed context

While setting up our experiments we realized that the head computer was not reachable, we also noticed that the head display wasn't turning on either. This is suggesting that the computer is not turning on.
So after three restarts of the whole robot and system, the same behavior re-occured.

I attach some pictures of the head:
![Selected photo (2)](https://github.com/robotology/icub-tech-support/assets/86918431/8f4f3e14-d63a-40cd-a9a3-110ea1a51bfc)
![Selected photo (1)](https://github.com/robotology/icub-tech-support/assets/86918431/2365a7ab-6171-4088-8f6c-4d7af7ef6437)
![Selected photo](https://github.com/robotology/icub-tech-support/assets/86918431/6c58eb0e-eb64-4969-b0cc-418abef2913f)

cc @mebbaid @vigisushrutha23 @S-Dafarra 

### How does it affect you?

We cannot proceed with our experiments, since we need to be able to access the lidar and camera

**Comment by AntonioConsilvio** [comment]
Hi @SimoneMic, after some testing, we realised that the problem was that Xavier was not completing the boot sequence.

Thanks to @davidelasagna's help, he was able to recover by refleshing the Xavier's internal memory.

Together with @martinaxgloria we downloaded everything we needed (superbuild and the repositories to use lidar and RealSense + all the necessary dependencies).

We are having some problems with lidar (see [issue](https://github.com/robotology/icub-tech-support/issues/1705)), but everything else is working properly, see RealSense:

![Screenshot from 2024-02-29 14-41-26](https://github.com/robotology/icub-tech-support/assets/114915464/5309ce9f-d453-4450-ba0c-4e402aa1c567)

Thank you all for your help!

cc @maggia80 @sgiraz @Fabrizio69 @S-Dafarra
